src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -251,7 +251,7 @@ class ValidatorStore extends BaseStore {
       const clusterHash = clusterStore.getClusterHash(payload[1]);
       const response = await Validator.getInstance().getClusterData(clusterHash);
       const process: RegisterValidator | SingleCluster = <RegisterValidator | SingleCluster>processStore.process;
-      const gasPrice = getRegisterValidatorGasPrice(!!response.cluster, payload[1].length, 'registerValidator' in process && process.registerValidator?.depositAmount);
+      const gasLimit = getRegisterValidatorGasLimit(!!response.cluster, payload[1].length, 'registerValidator' in process && process.registerValidator?.depositAmount <= 0);
```

Comment:
It is better to use selected operators length than payload[0] because in future payload could be changed in it's structure and it will cause bugs. 

This should be applied to all such places